src/patternfly/demos/DataToolbar/examples/data-toolbar-attribute-value-search-filter-example.hbs

```diff
@@ -0,0 +1,73 @@
+{{#> data-toolbar data-toolbar--id="data-toolbar-attribute-value-search-filter-example"}}
+  {{#> data-toolbar-content}}
+    {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-toggle-group pf-m-reveal-on-lg" data-toolbar-group--IsExpanded="false"}}
+      {{> data-toolbar-toggle}}
```

Comment:
This creates a button with `aria-controls="data-toolbar-attribute-value-search-filter-example-expandable-content"` but the expandable content for this example doesn't exist.